docs/specification/1.0.md

```diff
@@ -204,8 +205,8 @@ Each **Card** is described by the following attributes.
 Field | Description
 ----- | -----------
 `summary` | *string*. one-sentence, <140-character summary message for display to the user inside of this card.
-`detail` | *string*.  optional detailed information to display, represented in [(GitHub Flavored) Markdown](https://github.github.com/gfm/). (For non-urgent cards, the EHR may hide these details until the user clicks a link like "view more details...".) 
-`indicator` | *string*.  urgency/importance of what this card conveys. Allowed values, in order of increasing urgency, are: `info`, `warning`, `hard-stop`. The EHR can use this field to help make UI display decisions such as sort order or coloring. The value `hard-stop` indicates that the workflow should not be allowed to proceed. 
+`detail` | *string*.  optional detailed information to display, represented in [(GitHub Flavored) Markdown](https://github.github.com/gfm/). (For non-urgent cards, the EHR may hide these details until the user clicks a link like "view more details...".)
```

Comment:
You shouldn't have any changes on these lines.

Reply:
there are trailing whitespace in couple of lines in this file. Not sure why but the courtesy of atom whitespace package automagically removes them when saving the file based on package setting and I let it. But I have changed it back now to move forward with this PR.

Reply:
Thanks @yashaskram. fyi, if you use `git add --interactive` you can add just the changes you want in a file rather than the entire file. Also, I know some Git GUI programs support adding individual hunks as well.